presto-main/src/test/java/com/facebook/presto/execution/TestSqlStageExecution.java

```diff
@@ -173,7 +173,7 @@ public void testSplitAssignmentBatchSizeGreaterThanMaxPending()
 
         // The stage scheduler will block and this will cause a timeout exception
         try {
-            future1.get(2, TimeUnit.SECONDS);
+            future1.get(10, TimeUnit.SECONDS);
```

Comment:
If the timeout is expected. There might need to be a failure after `future1.get(10, TimeUnit.SECONDS);`?